src/lib.rs

```diff
@@ -45,17 +45,25 @@ impl Eta {
         Eta::create_instance(tasks_count, time_accuracy, tasks_done)
     }
 
-    /*fn pause(&self){
-        //tbd
+    pub fn pause(&mut self) {
```

Comment:
You need to account for time until ETA is paused, hence making paused a tuple `(Option<Instant>, usize)`, and adding the required implementations.